maskit/examples/load_data.py

```diff
@@ -0,0 +1,22 @@
+from maskit.examples.circles import circles
+from maskit.examples.iris import iris
+from maskit.examples.mnist import mnist
+
+
+def load_data(dataset, **kwargs):
+    train_size = kwargs.get("train_size", 100)
+    test_size = kwargs.get("test_size", 50)
+    shuffle = kwargs.get("shuffle", True)
+    classes = kwargs.get("classes", [6, 9])
+    wires = kwargs.get("wires", 4)
+
+    if dataset == "simple":
+        return [None, None, None, None]
```

Comment:
To make it easier for type annotations, you could event assign this to `Data`. But I think, you might even drop this and we might handle this by using `dataset=None` on the outside.

Reply:
I think removing the simple dataset would be a good idea, as it's not a real dataset. But I'm not sure about dropping it completely and checking on the outside, which would probably be the main.